model-optimizer/requirements.txt

```diff
@@ -1,11 +1,11 @@
-tensorflow>=1.15.2,<2.0; python_version < "3.8"
-tensorflow>=2.2; python_version >= "3.8"
-mxnet>=1.0.0,<=1.7.0
-networkx>=1.11
-numpy>=1.14.0,<1.19.0
-protobuf>=3.6.1
-onnx>=1.1.2
+tensorflow>=2.4.1
+mxnet>=1.2.0 ; sys_platform == 'win32'
+mxnet>=1.7.0.post2 ; sys_platform == 'linux'
+networkx~=2.5
+numpy~=1.19.5
+protobuf>=3.15.6
+onnx>=1.8.1
 test-generator==0.1.1
```

Comment:
Please, remove `test-generator=0.1.1` from all files except requirements_dev.txt. This change has been merged in the MO.